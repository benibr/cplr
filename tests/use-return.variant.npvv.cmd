../cplr 'ret = 69' -npvv
