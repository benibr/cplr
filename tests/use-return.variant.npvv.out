Preparation phase
Generation phase
Generating code
Generating labeled statements
Generated: 182 bytes code, 0 bytes dump
Compilation phase
Compilation succeeded
